README.md

```diff
@@ -1,5 +1,8 @@
 # Deep Learning in Julia
 
+[![Website link](https://img.shields.io/badge/URL-website-blue)](http://www.deep-learning.club/)
```

Comment:
You can actually use https here, even though it doesn't seem to be enforced when visiting the website for some reason.

Reply:
good idea, done!